php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -55,6 +57,7 @@
     Kind.TRY_STATEMENT,
     Kind.CATCH_BLOCK
   };
+  private static final Set<String> PHP_KEYWORD_VALUES = new HashSet<>(Arrays.asList(PHPKeyword.getKeywordValues()));
```

Comment:
This constant should probably be part of the tokenVisitor class.

Reply:
@benzonico Sure, I don't know why I didn't do it like that in the first place...